### PR TITLE
[log](insert-into) add log in parse insert into values data

### PR DESCRIPTION
**Pull request by sollhui**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

File: be/src/vec/exec/vunion_node.cpp [hunk 1]
```diff
@@ -229,12 +229,14 @@ Status VUnionNode::get_next_const(RuntimeState* state, Block* block) {
...
         }
         tmp_block.erase_not_in(result_list);
+        LOG(INFO) << "tmp_block rows: " << tmp_block.rows();
         if (tmp_block.rows() > 0) {
             RETURN_IF_ERROR(mblock.merge(tmp_block));
             tmp_block.clear();
         }
     }
     block->set_columns(std::move(mblock.mutable_columns()));
+    LOG(INFO) << "block rows: " << block->rows();
```
Comment (liaoxin01) [inline be/src/vec/exec/vunion_node.cpp:239]: can we print query id here?

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by yiguolei — DISMISSED** [review]

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by liaoxin01 — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**